W-18707073-slb traffic switching lds (#802)
mulesoft/docs-runtime-manager: cloudhub/modules/ROOT/pages/app-migration.adoc

.. Select whether you want to provide internal or external access to your endpoint.
. Follow the steps to xref:cloudhub-2::ch2-deploy.adoc[deploy an application to CloudHub 2.0].

[WARNING]
====

> If you use a different name for the CloudHub 2.0 application, make sure you input the CloudHub application name in the subdomain field of the endpoint configuration within the *Ingress* tab settings. Failing to configure the correct URLs can result in traffic not reaching the CloudHub 2.0 application during xref:app-migration.adoc#next-steps-after-application-upgrade[traffic switching].

Reviewer: ```suggestion
If you use a different name for the CloudHub 2.0 application, make sure you put the CloudHub application name in the *Ingress* tab settings in the subdomain field of the endpoint configuration. If you configure URLs incorrectly, traffic doesn't reach the CloudHub 2.0 application during xref:app-migration.adoc#next-steps-after-application-upgrade[traffic switching].
```

Reviewer: Simplified language, active vs. passive, and putting the procedural part in the order they'd encounter it. Please verify that is correct. :)

> By default, this action enables Dedicated Load Balancer (DLB) traffic switching. To enable Shared Load Balancer (SLB) traffic switching, add the CloudHub application endpoint in the format: `<app_name>.<region>.cloudhub.io`. For more information about SLB and DLB traffic switching, see xref:app-migration.adoc#understanding-the-traffic-source[Understanding the Traffic Source].

Reviewer: Do you need "For more information about SLB and DLB traffic switching" or could you just say "See ..." (SF CX standard) Or at least, "For more information, see"

Author: I would leave it as is, because the linked section does no elaborate on enabling SLB traffic switching, but just provides theoretical info on SLB and DLB traffic source and traffic switching.

Reviewer: Oh, gotcha. That makes sense then. Thanks for clarifying.

====

[NOTE]

> The *Host* dropdown list shows only the domains added under the certificates of the underlying VPC Dedicated Load Balancer configuration before the application upgrade started. If you add a new certificate to the underlying VPC Dedicated Load Balancer after the application upgrade has started, you need to manually add it under the *Domain & TLS* tab of your newly created private space for that domain to appear on the *Host* dropdown list.

Reviewer: underlying seems like an unusual word to use, but if that's standard for this doc, it's okay. Is it needed at all?
